components/cascader/demo/right-to-left.md

```diff
@@ -0,0 +1,89 @@
+---
+order: 12
+title:
+  zh-CN: 右到左
+  en-US: Right to left
+  fa-IR: چپ به راست
+---
+
+## zh-CN
+
+省市区级联。
+
+## en-US
+
+Cascader component supports right to left (rtl) direction if placed inside `ConfigProvider` with `direction='rtl'`
```

Comment:
This should also doc in `ConfigProvider` page.

Reply:
`ConfigProvider` documentations updated.